spec/lib/monitoring/middleware/prometheus_exporter_spec.rb

```diff
@@ -16,17 +16,21 @@
 
   let(:options) { { registry: registry, path: path} }
 
+  let(:env) { Rack::MockRequest.env_for }
+
   let(:app) do
-    app = ->(_) { [200, { 'Content-Type' => 'text/html' }, ['OK']] }
-    described_class.new(app, **options)
+    app = ->(env) { [200, { 'Content-Type' => 'text/html' }, ['OK']] }
   end
 
+  subject { described_class.new(app, **options) }
+
   context 'when requesting app endpoints' do
     it 'returns the app response' do
-      get '/foo'
+      env['PATH_INFO'] = "/foo"
+      status, _headers, _response = subject.call(env)
 
-      expect(last_response).to be_ok
-      expect(last_response.body).to eql('OK')
+      expect(status).to eql(200)
+      expect(_response.first).to eql('OK')
```

Comment:
Generally, a leading underscore indicates that the return value isn't used. Since we are using `_response` here, can we remove the underscore? (The same applies to other tests in this file as well)